src/state-machine/machine.ts

```
@@ -226,31 +74,16 @@ export class MeetingStateMachine {
     }
 
     public getError(): Error | null {
-        return this.error
+        return this.context.error || null
     }
 
     public getStartTime(): number {
         return this.context.startTime!
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)